apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx

```diff
@@ -123,11 +128,15 @@ const Layout = ({ category, cards }: LayoutProps) => {
       return (
         <Carousel
           items={items}
-          styles={{ widthFactor: cardsSorted[0].carouselWidthFactor || WidthFactor.Full }}
+          styles={{
+            widthFactor: cardsSorted[0].carouselWidthFactor || WidthFactor.Full,
+            pagination: category.hasPagination,
+            gap: cardsSorted[0].gridWidthFactor === WidthFactor.Full ? 6 : 8,
+          }}
         />
       );
     case ContentCardsLayout.grid:
-      return <Grid items={items} />;
+      return <Grid items={items} styles={{ widthFactor: cardsSorted[0].gridWidthFactor }} />;
```

Comment:
`cardsSorted` have all the same `gridWidthFactor` ? Just to understand correctly